desktop/app/main.js

```diff
@@ -1,18 +1,38 @@
 // This file is ES5; it's loaded before Babel.
-var app = require('app')
-var BrowserWindow = require('browser-window')
-var mainWindow = null
+require('babel/register')({
+  extensions: ['.desktop.js', '.es6', '.es', '.jsx', '.js']
+})
+
+const app = require('app')
+const ipc = require('ipc')
+const Window = require('./window')
+
+const trackerWindow = new Window('tracker', {
+  width: 500, height: 300,
+  resizable: false,
+  fullscreen: false,
+  frame: false
+})
 
-app.on('window-all-closed', function () {
-  app.quit()
+const launcherWindow = new Window('launcher', {
+  title: 'Keybase',
+  width: 200, height: 250,
+  x: 50, y: 50,
+  resizable: false,
+  fullscreen: false,
+  'use-content-size': true
 })
 
-app.on('ready', function() {
-  mainWindow = new BrowserWindow({width: 1600, height: 1200})
-  var filename = 'file://' + __dirname + '/../renderer/index.html'
-  mainWindow.loadUrl(filename)
-  mainWindow.openDevTools()
-  mainWindow.on('closed', function () {
-    mainWindow = null
-  })
+const mainWindow = new Window('index', {
+  width: 1600, height: 1200, openDevTools: true
 })
+
+app.on('ready', function () {
+  launcherWindow.show()
+  launcherWindow.window.on('closed', () => { app.quit() })
+
+  require('./notifications')
```

Comment:
I don't like a require having side effects. Feels weird (I know engine does it, but I don't think we should make it a habit)